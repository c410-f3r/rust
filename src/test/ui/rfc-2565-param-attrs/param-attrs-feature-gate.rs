// gate-test-param_attrs

#![deny(unused_variables)]

fn foo(
    /// Foo
    //~^ ERROR documentation comments cannot be applied to function parameters
    //~| NOTE doc comments are not allowed here
    #[allow(unused_variables)] a: u8
) {}

fn main() {}
