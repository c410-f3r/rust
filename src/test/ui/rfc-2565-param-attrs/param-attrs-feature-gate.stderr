error: documentation comments cannot be applied to function parameters
  --> $DIR/param-attrs-feature-gate.rs:6:5
   |
LL |     /// Foo
   |     ^^^^^^^ doc comments are not allowed here

error: aborting due to previous error

